css/selectors/focus-visible-011.html

```diff
@@ -37,30 +37,34 @@
   <ul id="instructions">
     <li>Click "Click here and press right arrow.".</li>
     <li>Press the right arrow key.</li>
-    <li>If "Focus moves here." has a red background, then the test result is FAILURE.
+    <li>If the element has a red background, then the test result is FAILURE.
         If it has a green outline, then the test result is SUCCESS.</li>
   </ul>
   <br />
-  <button id="start" tabindex="0">Click here and press right arrow.</button>
-  <button id="next" tabindex="-1">Focus moves here.</button>
+  <div id="target" tabindex="0">Click here and press right arrow.</div>
   <script>
-    start.addEventListener('keydown', (e) => {
+    target.addEventListener("focus", () => {
+      const arrow_right = "\ue014";
+      test_driver.send_keys(target, arrow_right);
+    });
+    target.addEventListener("keydown", (e) => {
+      e.preventDefault();
+    });
+    target.addEventListener("keyup", (e) => {
+      e.preventDefault();
+    });
+    target.addEventListener("keypress", (e) => {
       e.preventDefault();
-      next.focus();
     });
-
     async_test(function(t) {
-      next.addEventListener("focus", t.step_func(() => {
-        assert_equals(getComputedStyle(next).outlineColor, "rgb(0, 128, 0)", `outlineColor for ${next.tagName}#${next.id} should be green`);
-        assert_not_equals(getComputedStyle(next).backgroundColor, "rgb(255, 0, 0)", `backgroundColor for ${next.tagName}#${next.id} should NOT be red`);
-        t.done()
-      }));
+      target.addEventListener("keyup", t.step_func_done((e) => {
+        e.preventDefault();
```

Comment:
You already did `preventDefault()` above?

Reply:
Right, removing this.